modules/login/src/main/frontend/src/login/loginForm.js

```diff
@@ -145,114 +128,128 @@ class SignIn extends React.Component {
             let checkLoginTimer = setInterval(() => {
               if (loginPopup.closed === true) {
                 clearInterval(checkLoginTimer);
-                this.props.handleLogin && this.props.handleLogin(true);
+                handleLogin?.(true);
               }
             }, 1000);
           }
         })
-        .catch((err) => this.setState({failedLogin: "Error occurred while handling third-party identity provider"}))
+        .catch((err) => setFailedLogin("Error occurred while handling third-party identity provider"));
```

Comment:
While we're here, let's make this more english: 
```suggestion
        .catch((err) => setFailedLogin("An error occurred while handling the third-party identity provider."));
```